src/test/java/com/medeiros/RoostTest/archivePostTest.java

```diff
@@ -61,7 +61,7 @@ public void archivePost_Test() {
 					System.out.println("Description: Archive Restaurant Table Group");
       
               if (response.jsonPath().get("entries") != null) {  
-                MatcherAssert.assertThat(response.jsonPath().get("entries"), instanceOf(undefined.class));  
+                MatcherAssert.assertThat(response.jsonPath().get("entries"), is(nullValue()));  
```

Comment:
```
[ERROR] /C:/Program Files/Go/src/github.com/API-MusicBlender-Java/src/test/java/com/medeiros/RoostTest/archivePostTest.java:[64,89] cannot find symbol
[ERROR]   symbol:   class undefined
[ERROR]   location: class com.medeiros.RoostTest.archivePostTest
```

`undefined.Class` was used. There is no class of undefined, that's why got error.

**FIX** :- use `is(nullValue)` instead